examples/Calibration/Life_Cycle_example.py

```diff
@@ -0,0 +1,114 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py:percent
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.2'
+#       jupytext_version: 1.2.4
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+# %%
+from HARK.ConsumptionSaving.ConsIndShockModel import (
+    IndShockConsumerType,
+    init_lifecycle,
+)
+
+from HARK.Calibration.Income.IncomeTools import (
+    ParseIncomeSpec,
+    ParseTimeParams,
+    CGM_income,
+)
+
+from HARK.datasets.life_tables.us_ssa.SSATools import parse_ssa_life_table
+from HARK.datasets.SCF.WealthIncomeDist.SCFDistTools import income_wealth_dists_from_scf
+import matplotlib.pyplot as plt
+import pandas as pd
+from copy import copy
+
+# %% Alter calibration
+birth_age = 25
+death_age = 90
+adjust_infl_to = 1992
+income_calib = CGM_income
+education = "College"
+
+# Income specification
+income_params = ParseIncomeSpec(
+    age_min=birth_age,
+    age_max=death_age,
+    adjust_infl_to=adjust_infl_to,
+    **income_calib[education],
+    SabelhausSong=True
+)
+
+# Initial distribution of wealth and permanent income
+dist_params = income_wealth_dists_from_scf(
+    base_year=adjust_infl_to, age=birth_age, education=education, wave=1995
+)
+
+# We need survival probabilities only up to death_age-1, because survival
+# probability at death_age is 1.
+liv_prb = parse_ssa_life_table(
+    female=True, cross_sec=True, year=2004, min_age=birth_age, max_age=death_age - 1
+)
+
+# Parameters related to the number of periods implied by the calibration
+time_params = ParseTimeParams(age_birth=birth_age, age_death=death_age)
+
+# Update all the new parameters
+params = copy(init_lifecycle)
+params.update(time_params)
+params.update(dist_params)
+params.update(income_params)
+params.update({"LivPrb": liv_prb})
+
+# %% Create and solve agent
+Agent = IndShockConsumerType(**params)
+Agent.solve()
+
+# %% Simulation
+
+# Number of agents and periods in the simulation.
+Agent.AgentCount = 500
+Agent.T_sim = 200
+
+# Set up the variables we want to keep track of.
+Agent.track_vars = ["aNrmNow", "cNrmNow", "pLvlNow", "t_age", "mNrmNow"]
```

Comment:
Just FYI , this is going to conflict with PR #936, which I'm planning to do today.

Reply:
Thanks for the heads up!

Reply:
Fixed!